<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QMainWindow" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1283</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>39</y>
      <width>1251</width>
      <height>671</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QTabWidget" name="tabWidget_3">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_5">
        <attribute name="title">
         <string>Students</string>
        </attribute>
        <widget class="QTableView" name="tableView_students">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>1201</width>
           <height>231</height>
          </rect>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
        </widget>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>250</y>
           <width>931</width>
           <height>191</height>
          </rect>
         </property>
         <property name="title">
          <string>GroupBox</string>
         </property>
         <widget class="QLineEdit" name="lineEdit_degree_id">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>51</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_date_admitted">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>40</y>
            <width>71</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_first_name">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>40</y>
            <width>91</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_last_name">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>40</y>
            <width>91</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_phone">
          <property name="geometry">
           <rect>
            <x>350</x>
            <y>40</y>
            <width>91</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_email">
          <property name="geometry">
           <rect>
            <x>450</x>
            <y>40</y>
            <width>111</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_address">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>121</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_city">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>90</y>
            <width>81</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_state">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>90</y>
            <width>81</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_zip_code">
          <property name="geometry">
           <rect>
            <x>330</x>
            <y>90</y>
            <width>61</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_Add">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>160</y>
            <width>71</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Add Record</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_Update">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>160</y>
            <width>81</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Update Record</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>51</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>degree_id</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>20</y>
            <width>71</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>date_admitted</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>20</y>
            <width>51</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>first_name</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>270</x>
            <y>20</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>last_name</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>370</x>
            <y>20</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>phone</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>490</x>
            <y>20</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>email</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_Delete">
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>160</y>
            <width>81</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Delete Record</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_12">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>70</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>address</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_13">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>70</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>city</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_14">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>70</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>state</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_15">
          <property name="geometry">
           <rect>
            <x>340</x>
            <y>70</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>zip_code</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>400</x>
            <y>90</y>
            <width>71</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Clear Fields</string>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_6">
        <attribute name="title">
         <string>Assignments</string>
        </attribute>
        <widget class="QTableView" name="tableView_assignments_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>771</width>
           <height>251</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>281</width>
      <height>33</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Welcome, </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_admFname">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>fname</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1283</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLogout"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
